v2/bindings/csharp/SnapPluginLibCs/SnapPluginLib/CBridgeLinux.cs

```diff
@@ -0,0 +1,78 @@
+﻿using System;
+using System.Runtime.InteropServices;
+
+namespace SnapPluginLib
+{
+    /*
+     * Responsible for calling all exported (native) C functions 
+     */
+    internal static class CBridgeLinux
+    {
+        private const string PluginLibDllName = "swisnap-plugin-lib.so";
+
+        // Runner
+
+        [DllImport(PluginLibDllName, CharSet = CharSet.Ansi, SetLastError = true)]
+        internal static extern void start_collector(
+            Runner.CollectHandler collectHandler,
+            Runner.LoadHandler loadHandler,
+            Runner.UnloadHandler unloadHandler,
+            Runner.DefineHandler defineHandler,
+            string name,
+            string version
+        );
+
+        // Collect context related functions
+
+        [DllImport(PluginLibDllName, CharSet = CharSet.Ansi, SetLastError = true)]
```

Comment:
why not `CharSet.Unicode` ?

Reply:
because the current implementation of C bridge is to build on C ANSI